dex/msgjson/types.go

```diff
@@ -669,6 +690,7 @@ type OrderBook struct {
 	MarketID string `json:"marketid"`
 	Seq      uint64 `json:"seq"`
 	Epoch    uint64 `json:"epoch"`
+	// MarketStatus `json:"status"`// maybe
```

Comment:
Also here for consideration, but it should be redundant with `'config'` response data and the `TradeSuspension` + `TradeResumption` ntfns.